pruntime_rpc.proto

```diff
@@ -486,3 +492,56 @@ message HttpResponse {
   // The body payload
   bytes body = 3;
 }
+
+// Parameters for RPC GetContractInfo
+message GetContractInfoRequest {
+  // The contract to query about. Leave empty to query all.
+  string contract_id = 1;
```

Comment:
Should we just pass an array for multi-contract query?

Reply:
Sounds OK.

Reply:
Done